fairing/builders/cluster/gcs_context.py

```diff
@@ -54,7 +54,12 @@ def cleanup(self):
             self.created_secret.metadata.namespace,
             client.V1DeleteOptions())
 
-    def generate_pod_spec(self, image_name):
+    def generate_pod_spec(self, image_name, push):
+        args = ["--dockerfile=Dockerfile",
+                          "--destination=" + image_name,
+                          "--context=" + self.uploaded_context_url]
+        if not push:
+            args.append("--no-push")
```

Comment:
For no push case for cluster builder, where does the built image live? I am not able understand a valid use case for this.

Reply:
Testing/parity with other builders. It’s mainly functional for the docker daemon alone right now, but the append builder and cluster should optionally perform a docker load to have the same effect